tests/va_profile_delete_cron_job/test_va_profile_delete_cron_job.py

```diff
@@ -0,0 +1,78 @@
+import os
+from lambda_functions.va_profile_delete_cron_job.va_profile_delete_cron_job import va_profile_delete_cron_job_handler
+from sqlalchemy import text
+
+OPT_IN_OUT = text("""SELECT va_profile_opt_in_out(:va_profile_id, :communication_item_id, :communication_channel_id, :allowed, :source_datetime);""")
+
+DELETE_CRON_JOB_QUERY = text("""SELECT va_profile_delete_cron_job();""")
+
+COUNT = r"""SELECT COUNT(*) FROM va_profile_local_cache;"""
+
+#set up db connection
+
+def setup_db(connection):
+    """
+    Using the given connection, truncate the VA Profile local cache, and call the stored procedure to add a specific row.
+    This establishes a known state for testing.
+    Truncating is necessary because the database side effects of executing the VA Profile lambda function are not rolled
+    back at the conclusion of a test.
+    """
+
+    connection.execute("truncate va_profile_local_cache;")
```

Comment:
I don't think truncating is necessary for you because your lambda doesn't create rows.  You can probably delete this function entirely.l